src/internal.c

```diff
@@ -28070,7 +28069,16 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
         if (!session) {
             WOLFSSL_MSG("Session lookup for resume failed");
             ssl->options.resuming = 0;
+            return ret;
         }
+#if defined(HAVE_SESSION_TICKET) && !defined(WOLFSSL_NO_TICKET_EXPIRE)
+        /* check if the ticket is valid */
+        if (LowResTimer() > session->bornOn + ssl->timeout) {
+            WOLFSSL_MSG("Expired session ticket, fall back to full handshake.");
+            ssl->options.resuming = 0;
+        }
+#endif /* HAVE_SESSION_TICKET || !WOLFSSL_NO_TICKET_EXPIRE */
```

Comment:
This comment should be `&&` not `||`. Also please document the new macro `WOLFSSL_NO_TICKET_EXPIRE` above. You will also need to wrap this in `NO_ASN_TIME`, which disables `LowResTimer`.

Reply:
Fixed the comment and add NO_ASN_TIME macro guard. Added comment on top of this function about WOLFSSL_NO_TICKET_EXPIRE macro.